spec/person_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+require 'pry-byebug'
+require './lib/person.rb'
+require './lib/library.rb'
+
+describe Person do
+
+  subject {described_class.new}
```

Comment:
This line here is not necessary. RSpec will create an instance of the class under test and store it in `subject` for you. Do this if you want to override values that are set by default by the class constructor.